### PR TITLE
fix(AV-1743): Updated dataset sorting

### PR DESCRIPTION
**Pull request by teravill**

### CODE REVIEW

**Comment by teravill** [comment]
Reordered, removed and added filters to the dataset search according to ticket AV-1743

File: modules/ckanext-ytp_main/ckanext/ytp/plugin.py [hunk 1]
```diff
@@ -264,16 +263,16 @@ def dataset_facets(self, facets_dict, package_type):
...
+        facets_dict['vocab_keywords_' + lang] = _('Tags')
+        facets_dict.update({'organization': _('Organization')})
         facets_dict.update({'res_format': _('Formats')})
-        # BFW: source is not part of the schema. created artificially at before_index function
-        facets_dict.update({'source': _('Sources')})
         facets_dict.update({'license_id': _('Licenses')})
-        # add more dataset facets here
+        facets_dict.update({'groups': _('Category')})
         facets_dict.update({'producer_type': _('Producer type')})
+        # BFW: source is not part of the schema. created artificially at before_index function
```
Comment (Zharktas) [inline modules/ckanext-ytp_main/ckanext/ytp/plugin.py]: This comment could be removed as it refers to facet that no longer exists.

**Review by Zharktas — CHANGES_REQUESTED** [review]
Minor cleanup.

File: modules/ckanext-ytp_main/ckanext/ytp/plugin.py (hunk 1 above)
Comment (teravill) [reply modules/ckanext-ytp_main/ckanext/ytp/plugin.py]: Fixed in 324de87e3974505ecb310d562bffd1ebc486ae24

**Review by bzar — APPROVED** [review]

**Pull request merged**